src/module/actor/sheet/base.ts

```diff
@@ -589,6 +598,80 @@ abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorSheet<TActo
             ]);
         });
 
+        // Inventory sorting
+        const inventoryList = htmlQueryAll(html, "section.inventory-list, ol[data-container-type=actorInventory]").find(
+            (e) => e !== null
+        );
```

Comment:
Shouldn't this just be an `htmlQuery`? `htmlQueryAll` can't return an array with `null` elements.

Reply:
You're right. I was going from the assumption that two selectors needed to be resolved with ``htmlQueryAll` but `htmlQuery` works just fine.